js/index.js

```diff
@@ -19,24 +19,26 @@ for (i=0; i<skills.length; i++){
     skillsSection.appendChild(skill)
 }
 
-let githubRequest = new XMLHttpRequest ();
+
 const GITHUB_USERNAME = 'Asel-K'
-githubRequest.open("GET", `https://api.github.com/users/${GITHUB_USERNAME}/repos`);
-githubRequest.send(); 
-githubRequest.onload = function () {
-    let repositories = JSON.parse (this.response);
-    console.log ("response", repositories)
 
-let projectSection = document.getElementById ("projects");
-let projectList = projectSection.querySelector('ul')
-for(let i=0; i<repositories.length; i++){
-    let project = document.createElement('li');
-    project.innerHTML=`<a href=${`${repositories[i].clone_url}`}>${repositories[i].name}
-    </a>`
-    projectList.appendChild(project);
-}
-}
+//FETCHING API//
+fetch(`https://api.github.com/users/${GITHUB_USERNAME}/repos`)
+
+    .then(response=> response.json())
+    .then(data=>repo(data))
```

Comment:
Your code works great, and fulfills all the requirements! For readability, I think this code block would be served well by cleaner white spacing, bracketing, and scoping. As an outside developer, I might ask myself, is `repo` meant to be used by functions outside of this `fetch` function? Where does the chaining of `then` methods end, ultimately? For example your `fetch` could look something like this to be more readable:

```javascript
//FETCHING API//
function repo(data) {
    let projectSection = document.getElementById ("projects");
    let projectList = projectSection.querySelector('ul')
    for(let i = 0; i < data.length; i++) {
        let project = document.createElement('li');
        project.innerHTML=`<a href=${`${data[i].clone_url}`}>${data[i].name} ${data[i].description}</a>`
        projectList.appendChild(project)
    }
}

fetch(`https://api.github.com/users/${GITHUB_USERNAME}/repos`)
    .then(response => response.json())
    .then(data => repo(data));
```